dev/com.ibm.websphere.org.eclipse.microprofile/bnd.bnd

```diff
@@ -0,0 +1,24 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion: 1.0
+
+-sub: *.bnd
+
+publish.wlp.jar.suffix: dev/api/stable
+
+instrument.disabled: true
+
+-buildpath: \
+  com.ibm.websphere.javaee.cdi.1.2;version=latest, \
+  com.ibm.websphere.javaee.interceptor.1.2;version=latest, \
+  com.ibm.websphere.javaee.jsonp.1.0, \
+  org.eclipse.microprofile.reactive.messaging:microprofile-reactive-messaging-api;version=1.0.0
```

Comment:
What does this root bnd.bnd file in the `com.ibm.websphere.org.eclipse.microprofile` project do?

Initially I expected it to be doing nothing, but there's quite a lot in here if that's the case.

Reply:
The outer project references the sub bundles.  The subprojects inherit settings from the outer bnd file.  The settings in the file (instrument.disabled, publish.wlp.jar.suffix and buildpath) are set for all of the sub projects.  Unless assemble.gradle is updated to handle some of these liberty build properties like instrument.disabled and publish.wlp.jar.suffix in the sub projects, they need to be in the outer bnd file.  The buildpath is defined by bnd and is required to be in the outer bnd file.  buildpath will not be used if it is specified in the sub bnd files.  In the end, the buildpath here is used to set the import versions in the sub project bundles.

Reply:
I will add some comments in the bnd file to help explain this.

Reply:
I updated the bnd.bnd files for the outer projects with comments.